lib/compat/wordpress-6.2/get-global-styles-and-settings.php

```diff
@@ -184,6 +184,41 @@ function gutenberg_get_global_settings( $path = array(), $context = array() ) {
 }
 
 /**
+ * Repeated logic from `get_default_block_editor_settings` function. When implemented into core,
+ * remove logic from `get_default_block_editor_settings` and simple call this function instead.
+ *
+ * @return array
+ */
+function gutenberg_get_block_theme_supports() {
+	$theme_settings = array(
+		'disableCustomColors'    => get_theme_support( 'disable-custom-colors' ),
```

Comment:
Looking at the code that uses [this data](https://github.com/oandregal/wordpress-develop/blob/ff7fd1e3932ad7b5ec8cd8cfa66ab6054e2de165/src/wp-includes/class-wp-theme-json.php#L2791) (I didn't find a more recent definition of this code in Gutenberg), we need:

- `disableCustomColors`
- `disableCustomGradients`
- `disableCustomFontSizes`
- `enableCustomLineHeight`
- `enableCustomUnits`
- `enableCustomSpacing`

It seems we can get rid of `disableLayoutStyles` in this method?